src/functions/addProvisionalTechRecord.ts

```diff
@@ -16,7 +16,9 @@ const addProvisionalTechRecord = async (event: any) => {
   if (!techRec || !techRec.length) {
     return Promise.resolve(new HTTPResponse(400, formatErrorMessage("Body is not a valid TechRecord")));
   }
-
+  if (techRec.length > 1) {
+    return Promise.resolve(new HTTPResponse(400, "Can only add one provisional record"));
```

Comment:
If i'm not mistaken, it is possible to have a technical record with 2 provisional records but no more should be allowed. Will this change prevent that from happening?

Reply:
This is to stop a client from sending multiple technical records in the technical record array when adding a provisional record. Should not be messing around with any of the existing business logic around having two provisional records.